tools/integration_tests/interrupt/git_clone_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Provides integration tests for symlink operation on local files.
+
+package interrupt
+
+import (
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/operations"
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/setup"
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/test_setup"
+)
+
+var (
+	testDirPath string
+)
+
+type ignoreInterruptsTest struct{}
+
+func (s *ignoreInterruptsTest) Teardown(t *testing.T) {}
+
+func (s *ignoreInterruptsTest) Setup(t *testing.T) {
+	testDirPath = setup.SetupTestDirectory(testDirName)
+}
+
+func (s *ignoreInterruptsTest) TestGitClone(t *testing.T) {
```

Comment:
wdyt, should we also add two more tests - git checkout and commit ? Asking because from a customer POV, they would expect git to work and not just git clone and it's like we got issue in git clone only so we didn't think further. Also, it's not possible to test git command, so probably 2-3 common (clone, checkout, commit) should suffice.

Reply:
Makes sense, added more tests. Thanks!